.idea/copyright/MCreator__templates_.xml

```diff
@@ -1,6 +1,6 @@
+    <option name="notice" value="MCreator (https://mcreator.net/)&#10;Copyright (C) 2012-2020, Pylo&#10;Copyright (C) 2020-2024, Pylo, opensource contributors&#10;&#10;This program is free software: you can redistribute it and/or modify&#10;it under the terms of the GNU General Public License as published by&#10;the Free Software Foundation, either version 3 of the License, or&#10;(at your option) any later version.&#10;&#10;This program is distributed in the hope that it will be useful,&#10;but WITHOUT ANY WARRANTY; without even the implied warranty of&#10;MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the&#10;GNU General Public License for more details.&#10;&#10;You should have received a copy of the GNU General Public License&#10;along with this program.  If not, see &lt;https://www.gnu.org/licenses/&gt;.&#10;&#10;Additional permission for code generator templates (*.ftl files)&#10;&#10;As a special exception, you may create a larger work that contains part or &#10;all of the MCreator code generator templates (*.ftl files) and distribute &#10;that work under terms of your choice, so long as that work isn't itself a &#10;template for code generation. Alternatively, if you modify or redistribute &#10;the template itself, you may (at your option) remove this special exception, &#10;which will cause the template and the resulting code generator output files &#10;to be licensed under the GNU General Public License without this special &#10;exception." />
     <option name="myName" value="MCreator (templates)" />
   </copyright>
-</component>
\ No newline at end of file
+</component>
```

Comment:
Undo newline addition here, applies to other files too, as otherwise IDEA will add it back in all other PRs